contracts/accounts/Simple7702Account.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8;
+pragma solidity ^0.8.28;
```

Comment:
I don't think it is a good idea to "shotgun" all sources wiht exact minor version we use. better stick with "major" , that is"0.8"
(the exception is EntryPoint itself, where the actual sub-version is important)

Reply:
(that is, I do think we need most of this refactor - but remove the minor subversion, instead of updating it to the current latest)

Reply:
@drortirosh 
That is the complete opposite of what we need. Solidity is actually making some pretty important, breaking changes in "minor" versions, and we do rely on them a lot. Our code will not compile with older versions.
Also, this is not a code that has to be imported and inherited in other projects, but only used when deployed on-chain.
We MUST specify the exact solidity version we are using, 100%.